<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Toynote</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/appicon.svg</normaloff>:/icons/appicon.svg</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTableView" name="notesView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>382</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashLine</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QCheckBox" name="checkBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Disable Note list</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Notebook"/>
    <addaction name="actionOpen_Notebook"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_As_Text"/>
    <addaction name="separator"/>
    <addaction name="actionCloseNotebook"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionVisit_eCourses"/>
    <addaction name="actionLottery"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Notebook"/>
   <addaction name="actionNew_Notebook"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Note"/>
   <addaction name="actionDelete_Notes"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Notebook">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-new.svg</normaloff>:/icons/Notebook-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New Notebook</string>
   </property>
   <property name="toolTip">
    <string>Create a new notebook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_Note">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Note-add.svg</normaloff>:/icons/Note-add.svg</iconset>
   </property>
   <property name="text">
    <string>New Note</string>
   </property>
   <property name="toolTip">
    <string>Add a new note</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionDelete_Notes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Note-remove.svg</normaloff>:/icons/Note-remove.svg</iconset>
   </property>
   <property name="text">
    <string>Delete Notes</string>
   </property>
   <property name="toolTip">
    <string>Delete the selected note(s)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/about.svg</normaloff>:/icons/about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About Toynote</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-save.svg</normaloff>:/icons/Notebook-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the notebook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Notebook">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-open.svg</normaloff>:/icons/Notebook-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Notebook</string>
   </property>
   <property name="toolTip">
    <string>Open a notebook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseNotebook">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close the notebook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-save-as.svg</normaloff>:/icons/Notebook-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve As</string>
   </property>
   <property name="toolTip">
    <string>Save the notebook to a chosen file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_As_Text">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-save-as.svg</normaloff>:/icons/Notebook-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save As &amp;Text</string>
   </property>
   <property name="toolTip">
    <string>Save the notebook as text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionVisit_eCourses">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/web.svg</normaloff>:/icons/web.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Visit eCourses</string>
   </property>
   <property name="toolTip">
    <string>Open the eCourses web page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/exit.svg</normaloff>:/icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLottery">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/lottery.svg</normaloff>:/icons/lottery.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Lottery</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/qt.svg</normaloff>:/icons/qt.svg</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Notebook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseNotebook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Notebook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveNotebookAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Note</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Notes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteNotes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVisit_eCourses</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>toCourses()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLottery</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>startLottery()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As_Text</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveNotebookAsText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notesView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>editNote(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>disableNoteList(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Qt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutqt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayAbout()</slot>
  <slot>saveNotebook()</slot>
  <slot>openNotebook()</slot>
  <slot>newNotebook()</slot>
  <slot>closeNotebook()</slot>
  <slot>saveNotebookAsText()</slot>
  <slot>newNote()</slot>
  <slot>deleteNotes()</slot>
  <slot>toCourses()</slot>
  <slot>exit()</slot>
  <slot>startLottery()</slot>
  <slot>saveNotebookAs()</slot>
  <slot>editNote(QModelIndex)</slot>
  <slot>disableNoteList(bool)</slot>
  <slot>aboutqt()</slot>
 </slots>
</ui>
